sphinx_rtd_theme/locale/sphinx.pot

```diff
@@ -0,0 +1,162 @@
+"Generated-By: Babel 2.5.3\n"
+
+#: sphinx_rtd_theme/breadcrumbs.html:31
+msgid "Docs"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:43 sphinx_rtd_theme/breadcrumbs.html:45
+msgid "Edit on GitHub"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:50 sphinx_rtd_theme/breadcrumbs.html:52
+msgid "Edit on Bitbucket"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:57 sphinx_rtd_theme/breadcrumbs.html:59
+msgid "Edit on GitLab"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:62 sphinx_rtd_theme/breadcrumbs.html:64
+msgid "View page source"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:74 sphinx_rtd_theme/footer.html:5
+msgid "Next"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:77 sphinx_rtd_theme/footer.html:8
+msgid "Previous"
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:19
+#, python-format
+msgid "&copy; <a href=\"%(path)s\">Copyright</a> %(copyright)s."
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:21
+#, python-format
+msgid "&copy; Copyright %(copyright)s."
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:26
+#, python-format
+msgid ""
+"\n"
+"        <span class=\"build\">\n"
+"          Build\n"
+"          <a href=\"%(build_url)s\">%(build_id)s</a>.\n"
+"        </span>\n"
+"      "
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:33
+#, python-format
+msgid ""
+"\n"
+"        <span class=\"commit\">\n"
+"          Revision <code>%(commit)s</code>.\n"
+"        </span>\n"
+"      "
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:39
+#, python-format
+msgid "Last updated on %(last_updated)s."
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:46
+msgid ""
+"Built with <a href=\"http://sphinx-doc.org/\">Sphinx</a> using a <a "
+"href=\"https://github.com/rtfd/sphinx_rtd_theme\">theme</a> provided by "
+"<a href=\"https://readthedocs.org\">Read the Docs</a>"
+msgstr ""
+
+#: sphinx_rtd_theme/layout.html:37
+#, python-format
+msgid "Search within %(docstitle)s"
+msgstr ""
+
+#: sphinx_rtd_theme/layout.html:61
+msgid "About these documents"
+msgstr ""
+
+#: sphinx_rtd_theme/layout.html:64
+msgid "Index"
+msgstr ""
+
+#: sphinx_rtd_theme/layout.html:67 sphinx_rtd_theme/search.html:11
+msgid "Search"
+msgstr ""
+
+#: sphinx_rtd_theme/layout.html:70
+msgid "Copyright"
+msgstr ""
+
+#: sphinx_rtd_theme/layout.html:107
+msgid "Logo"
+msgstr ""
+
+#: sphinx_rtd_theme/search.html:26
+msgid ""
+"Please activate JavaScript to enable the search\n"
+"      functionality."
+msgstr ""
+
+#: sphinx_rtd_theme/search.html:33
+msgid "Search Results"
+msgstr ""
+
+#: sphinx_rtd_theme/search.html:35
+msgid ""
+"Your search did not match any documents. Please make sure that all words "
+"are spelled correctly and that you've selected enough categories."
+msgstr ""
+
+#: sphinx_rtd_theme/searchbox.html:4
+msgid "Search docs"
+msgstr ""
+
+#: sphinx_rtd_theme/versions.html:11
+msgid "Versions"
+msgstr ""
+
+#: sphinx_rtd_theme/versions.html:17
+msgid "Downloads"
+msgstr ""
+
+#: sphinx_rtd_theme/versions.html:23
+msgid "On Read the Docs"
```

Comment:
@agjohnson "On Read the Docs" in Spanish has translation like "En Read the Docs" for that I believe that string can be translate!

Reply:
Correct, "En Read the Docs" is just fine. I think we just need a translator note here (again, I'm not familiar with babel) similar to how this is handled in with django's support:
https://docs.readthedocs.io/en/latest/i18n.html#localization-comments

Reply:
@agjohnson I used Babel because "Sphinx uses Babel to extract messages and maintain the catalog files." check out the following link http://www.sphinx-doc.org/en/master/devguide.html#locale-updates 

I trying to improvement the i18n support to the "sphinx_rst_theme" package because I am Plone Developer and Trainer, and time ago the official Plone training iniciative usa the "sphinx_rst_theme" package as theme into the training HTML recourses. Then I am working to translate all this training to Spanish but the "sphinx_rst_theme" package have some HTML template that don't have support for i18n. 

This is the main reason for my improvements of "sphinx_rst_theme" package, my logic said me that Sphinx using babel for extract and manage the string to translate, thene I need to adopts the same domain "sphinx" and the same library for extract and manage the new translation inside this theme package.

Now yo can undestand me why don't use any integration with Django, because I need use the "sphinx_rst_theme" package has a Sphinx theme for generate HTML documentations initialy.

Reply:
Oh sorry, I don't mean to say we should use Django's -- that certainly wouldn't work. I'm just not familiar with Babel as a tool yet. I was just noting that in Django, I have the ability to add some context for translators to the string to be translated. I'm sure Babel has something comparable, I'm just not sure what it is.